config/routes.rb

```diff
@@ -1,19 +1,19 @@
+
 Rails.application.routes.draw do
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-   root 'welcome#index'
-
+   #root 'welcome#index'
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
 
   # Example of named route that can be invoked with purchase_url(id: product.id)
   #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
-
+ get 'cards/test' => 'cards#test'
+ post 'cards/:id/test' => 'cards#proverka'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.